.travis.yml

```diff
@@ -96,6 +96,10 @@ install:
     - python setup.py install
     - python -c "import mne; mne.sys_info()"
     - python -c "import numpy;print(numpy.__version__)"
+    - python -c "import scipy;print(scipy.__version__)"
+    - python -c "import pandas;print(pandas.__version__)"
```

Comment:
Yes, I know :) but it seemed to me to be for debugging purposes and it is hard to debug when debugging statements raise errors. Anyway, the failing build and the one that was green is only a difference of printing numpy version - can these failing tests (`Test LCMV with evoked data and single trials` and `Test stc methods lh_data, rh_data, bin, center_of_mass, resample`) differ so much from one travis run to the next?